db/models.py

```diff
@@ -1 +1,26 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(unique=True)
+    bio = models.TextField(blank=True)
+    race = models.ForeignKey(Race, on_delete=models.PROTECT)
```

Comment:
it's better to use CASCADE because PROTECT has some disadvantages like you won't be able to delete race(for ex. it's canceled) because some players exist